exercises/go-counting/canonical-data.json

```diff
@@ -0,0 +1,172 @@
+{
+  "exercise": "go-counting",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Test for black territory on 5x5 board",
+      "property": "territory",
+      "input": {
+        "board": [
+		    "  B  ",
+		    " B B ",
+		    "B W B",
+		    " W W ",
+		    "  W  "
+		],
+        "x": 0,
+        "y": 1
+      },
+      "expected": {
+        "stone": BLACK
```

Comment:
`BLACK` is probably hi-lighted because it should probably be in `"`'s and should have a trailing `,` on the line